src/server-entry.ts

```diff
@@ -1,32 +1,32 @@
 import buildApp from '@/common/buildApp';
-import { config } from './server/TermboxConfig';
 import { factory } from './common/TermboxFactory';
 import WikibaseApiLanguageRepository from './server/data-access/WikibaseApiLanguageRepository';
 import MwBotWikibaseRepo from './server/data-access/MwBotWikibaseRepo';
 import mwbot from 'mwbot';
 import EntityInitializer from './common/EntityInitializer';
 import BundleBoundaryPassingException, { ErrorReason } from '@/server/exceptions/BundleBoundaryPassingException';
-import TermboxRequest from '@/common/TermboxRequest';
 import EntityNotFound from '@/common/data-access/error/EntityNotFound';
+import SsrRequest from './server/SsrRequest';
 
-const apiBot = new mwbot( {
-	apiUrl: config.getWikibaseRepoApi(), // TODO test
-} );
+export default ( ssrRequest: SsrRequest ) => {
```

Comment:
I'm not strongly opposing, but it does not sound quite right to make the config part of the request. It also means that any service dependent on its values can only be created within this function, which I find icky considering earlier today we said that `mwbot` looks like a good candidate to be shared between requests. Only having the API URL available here wouldn't make it impossible, but likely more complicated.

Reply:
While I agree with you that having the config appearing to be part of the request (maybe poor naming contributed to this), it is not quite what's happening per se.
As I understand it, please correct me, there is only one way to get information (_context_) into the bundle - through the first parameter of `serverBundle`'s `renderToString` method (that is src/server-entry.ts).
Consequently, if we _want_ to pass information into the bundle it has to go through here. Do you see an alternative?